torchbenchmark/e2e_models/hf_t5/__init__.py

```diff
@@ -22,6 +25,7 @@
     MBartTokenizer,
     MBartTokenizerFast
 )
+from transformers.models.t5.modeling_t5 import T5Block
```

Comment:
Just curious, what does this class do?

Reply:
T5block is the main building block of a T5 model that you stack. It's similar to a TransformerEncoderLayer in BERT. We use it in transformer_auto_wrap_policy, I believe to decide the sharding strategy to be every T5block layer (this I'm guessing).

Reply:
> LGTM, maybe we should attach the metrics result of 8 gpus 1 node to the PR body so that users get an idea what the output looks like.

You mean in the code in __init__.py? I have example results above.